src/utils/computeFiatValuePriceImpact.tsx

```diff
@@ -9,6 +9,10 @@ export function computeFiatValuePriceImpact(
   if (!fiatValueOutput || !fiatValueInput) return undefined
   if (!fiatValueInput.currency.equals(fiatValueOutput.currency)) return undefined
   if (JSBI.equal(fiatValueInput.quotient, JSBI.BigInt(0))) return undefined
-  const pct = ONE_HUNDRED_PERCENT.subtract(fiatValueOutput.divide(fiatValueInput))
+  // Example when output is greater than input:
+  // 1 - (100 / 80) = 1 - 1.25 = -0.25 = 25% profit (positive)
+  // Example when output is less than input:
+  // 1 - (80 / 100) = 1 - 0.8 = 0.2 = 20% loss (negative)
+  const pct = ONE_HUNDRED_PERCENT.subtract(fiatValueOutput.divide(fiatValueInput)).multiply(-1)
```

Comment:
this makes more sense if we flip the sign at the end

Reply:
<img width="394" alt="image" src="https://user-images.githubusercontent.com/59578595/226075895-9ade3e4d-b326-44aa-8c57-c12c09c8e0d3.png">
dont think this is right